### PR TITLE
Add some additional output for test failures

### PR DESCRIPTION
**Pull request by rohansingh**
This will help debug common test failures.

### CODE REVIEW

**Comment by rohansingh** [comment]
@davidxia

File: helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java [hunk 1]
```diff
@@ -300,6 +300,9 @@ public void addJob(final Job job) throws JobExistsException {
         }
         throw new JobExistsException(id.toString());
       }
+    } catch (NoNodeException e) {
```
Comment (davidxia) [inline helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java]: It's the `client.transaction()` that throws the `NoNodeException` right? How can this happen with `ensurePath()`?

Comment (rohansingh) [inline helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java]: If the /config or /config/jobs paths haven't been created for some reason then the `create` ops in the transaction above will throw.

**Comment by gimaker** [comment]
:+1:

File: helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java (hunk 1 above)
Comment (davidxia) [inline helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java]: Should we add an `ensurePath()` for those paths as well? Does this happen when the master is slow to startup?

**Pull request merged**